apps/web/src/components/pizza-formel.tsx

```diff
@@ -0,0 +1,18 @@
+type Props = {
+  registration_count: number;
+};
+
+export function PizzaFormel({ registration_count }: Props) {
+  if (registration_count < 0) {
+    return <p>Antall kan ikke være negativt</p>;
+  }
+  if (registration_count === 0) {
+    return <p>Ingen pizza</p>;
+  }
+  if (Number.isNaN(registration_count)) {
+    return <p>Input må være et tall</p>;
+  }
+
+  const pizza_count = registration_count * (3 / 8);
```

Comment:
Den offisielle formelen sier at man skal runde opp.

```suggestion
  const pizza_count = Math.ceil(registration_count * (3 / 8));
```